### PR TITLE
Creating Guidelines testing for user facing

### PR DESCRIPTION
**Pull request by MRotterAAN**

### CODE REVIEW

File: tests/Guidelines/guidelines.spec.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    // Go to https://www.aan.com/practice/guidelines
+    await page.goto(APP_URLS.GUIDELINES);
+  
+    // Click text=Learn more about Guidelines
+    await page.locator('text=Learn more about Guidelines').click();
+    await expect(page).toHaveURL(APP_URLS.AANDOTCOM + '/practice/what-are-clinical-practice-guidelines');
+  
+    // Click text=Guidelines FAQ Q. What is the difference between a practice guideline and a qual >> span >> nth=1
+    await page.locator('text=Guidelines FAQ Q. What is the difference between a practice guideline and a qual >> span').nth(1).click();
```
Comment (treither-aan) [inline tests/Guidelines/guidelines.spec.js:15]: Would it be possible to find a more succinct locator for this? Perhaps targeting the footer element inside the element with the class faq-drawer. I think this one will break if the text for the FAQs change.

Comment (treither-aan) [reply tests/Guidelines/guidelines.spec.js:15]: I know this wasn't mentioned in the test description, but would it be possible to take a screenshot here, after expanding the faq drawer?

File: tests/Guidelines/guidelines.spec.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  
+    // Fill [placeholder="Search Guidelines"]
+    await page.locator('input[name="GuidelinesSearchTerm"]').fill('Migraine');
+  
+    // Press Enter
+    await page.locator('input[name="GuidelinesSearchTerm"]').press('Enter');
+    await expect(page).toHaveURL(APP_URLS.AANDOTCOM + '/Guidelines/Home/Search');
+  
+    // Click text=Update: Pharmacologic Treatment for Episodic Migraine Prevention in Adults Publi
+    let searchText = await page.locator('text= Migraine ').first().innerText();
```
Comment (treither-aan) [inline tests/Guidelines/guidelines.spec.js:46]: Can this be updated to "h1 >> text=Migraine"? I think that would make sure that the locator is targeting the header specifically.

Comment (MRotterAAN) [reply tests/Guidelines/guidelines.spec.js:46]: So it has to be an a as the headers are a links so it works with that!

**Pull request closed without merging**